presto-main/src/main/java/com/facebook/presto/sql/planner/TranslationMap.java

```diff
@@ -41,6 +45,7 @@
     // all expressions are rewritten in terms of fields declared by this relation plan
     private final RelationPlan rewriteBase;
     private final Analysis analysis;
+    private final IdentityHashMap<LambdaArgumentDeclaration, Symbol> lambdaDeclarationToSymbolMap;
```

Comment:
Why is this separate from `expressionToSymbols`? Also, why isn't it copied in `copyMappingsFrom` and `putExpressionMappingsFrom`?